### PR TITLE
Add gene annotation manipulation CLI package

### PR DESCRIPTION
**Pull request by pinin4fjords**
finally added a conda package for this script, which I've used in various forms in multiple places. Those places can now be switched to use this.

https://github.com/ebi-gene-expression-group/atlas-gene-annotation-manipulation

### CODE REVIEW

**Review by pcm32 — APPROVED** [review]
LGTM

**Comment by pcm32** [comment]
Ahhh! one thing... are the exec excutable?

File: recipes/atlas-gene-annotation-manipulation/build.sh [hunk 1]
```diff
@@ -0,0 +1,2 @@
+mkdir -p $PREFIX/bin
+cp *.sh *.R $PREFIX/bin
```
Comment (pcm32) [inline recipes/atlas-gene-annotation-manipulation/build.sh:2]: are all these guys chmoded to be executable?

Comment (pinin4fjords) [reply recipes/atlas-gene-annotation-manipulation/build.sh:2]: Yep!

**Pull request merged**